js/user-form.js

```diff
@@ -1,24 +1,47 @@
-const getActivePage = (active) => {
 
-  if (active === false) {
-    const form = document.querySelector('.ad-form');
-    const formAvatar = form.querySelector('.ad-form-header__input');
-    const formFieldset = form.querySelector('.ad-form__element');
-    const mapForm = document.querySelector('.map__filters');
-    const mapFormFieldset = mapForm.querySelector('.map__filter');
+const addAttr = (formField) => {
```

Comment:
Если мы работаем с коллекцией элементов то нейминг явно должен нам об этом сообщать
```suggestion
const addAttr = (formFields) => {
```